### PR TITLE
Bubble Sort

### PR DESCRIPTION
**Pull request by athioushranjan**

### CODE REVIEW

File: go/programs/bubble_sort.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package programs
+
+import "fmt"
+
+func BubbleSort(array []int) []int {
```
Comment (GokuDBZ) [inline go/programs/bubble_sort.go]: Give some meaning full name to variable not just array

**Review by GokuDBZ — CHANGES_REQUESTED** [review]

File: go/programs/bubble_sort.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	size := len(array)-1
+	var iteration, swap int
+	for _= range array{
+		swap = 0
+		for index:= 1; index <= size-iteration; index++{
+			if array[index-1] > array[index]{
+				swap++
+				temp := array[index]
+				array[index] = array[index-1]
+				array[index-1] = temp
```
Comment (GokuDBZ) [inline go/programs/bubble_sort.go]: create a helper method to swap variables.

**Review by GokuDBZ — CHANGES_REQUESTED** [review]

File: go/programs/bubble_sort.go (hunk 2 above)
Comment (athioushranjan) [reply go/programs/bubble_sort.go]: okayy

File: go/programs/bubble_sort.go (hunk 1 above)
Comment (athioushranjan) [reply go/programs/bubble_sort.go]: done

**Pull request merged**